hotsos.sh

```diff
@@ -332,9 +352,13 @@ generate_summary ()
     else
         repo_info=`get_git_rev_info` || repo_info="unknown"
     fi
-    echo "hotsos:" > $MASTER_YAML_OUT
-    echo "  version: ${SNAP_REVISION:-"development"}" >> $MASTER_YAML_OUT
-    echo "  repo-info: $repo_info" >> $MASTER_YAML_OUT
+
+    cat <<EOF >> "${MASTER_YAML_OUT}"
+---
+hotsos:
+  version: ${SNAP_REVISION:-"development"}
+  repo-info: $repo_info
+EOF
```

Comment:
necessary? If so can we lose the ---

Reply:
Fixed (removed the `---`)

Reply:
this is not the same as the original code as it uses >> as opposed to >. Is there a need for this change? If there isnt can we remove it?

Reply:
Oh you are right. Will fix in next iteration.